pyannote/audio/core/model.py

```diff
@@ -270,9 +234,6 @@ def setup(self, stage=None):
             # setup custom validation metrics
             self.task.setup_validation_metric()
 
-            # cache for later (and to avoid later CUDA error with multiprocessing)
-            _ = self.example_output
-
```

Comment:
@clement-pages at least this line should no longer be a problem.

Reply:
I'll test with these changes and let you know.

Reply:
The problem of sending the model to the GPU is fixed, I update my tutorial PR to remove manual  sending.